README.md

```diff
@@ -369,6 +372,15 @@ crypto.high_dollar #'$3,590'
 
 See [#crypto](https://iexcloud.io/docs/api/#crypto) for detailed documentation or [crypto.rb](lib/iex/resources/crypto.rb) for returned fields.
 
+### Other Requests
+
+Public endpoints that aren't yet supported by the client can be called using `client.get`, `client.post`, `client.put` 
+and `client.delete` methods. For endpoints that require secret token it must be passed as an option explicitly:
```

Comment:
Tokens are required for all custom endpoints now, say: "Pass the required token explicitly."